dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/index/IndexServiceBean.java

```diff
@@ -896,6 +905,17 @@ private String addOrUpdateDataset(IndexableDataset indexableDataset) {
                             if (dsfSolrField.isFacetable()) {
                                 solrInputDocument.addField(solrFieldFacetable, htmlFreeText);
                             }
+                        } else if (FieldType.PERIODO.equals(dsfType.getFieldType())) {
+                            final String url = dsf.getValue();
+                            solrInputDocument.addField(PERIODO_ID, url);
+                            PeriodoDictionary.getByUrl(url).ifPresent(period -> {
+                                solrInputDocument.addField(PERIODO_LABEL, period.getLabel());
+                                solrInputDocument.addField(PERIODO_START, period.getStart());
+                                solrInputDocument.addField(PERIODO_STOP, period.getStop());
+                                solrInputDocument.addField(PERIODO_AUTHORITY_TITLE, period.getAuthorityTitle());
+                                solrInputDocument.addField(PERIODO_COVERAGE_NAME, period.getCoverageName());
+                                solrInputDocument.addField(PERIODO_LOCATIONS, period.getLocations());
```

Comment:
Not sure that we gain anything by indexing those fields. They won't be included when searching by simple query (they are not included in "catch all" field in schema, see: `<copyField ... />` in `schema.xml`). They won't be included in any facets. They won't be included in advanced search.